tests/integration/queries/test_workflow.py

```diff
@@ -515,12 +543,22 @@ def _new_dataset_for_updating(client):
         )
     )
 
+    questionnaire = client.call(
+        GetQuestionnaire(
+            questionnaire.model_group_by_name(
+                classifier_name
+            ).model_group.questionnaire_id
+        )
+    )
+    while questionnaire.questions_status == "STARTED":
```

Comment:
I think GetQuestionnaire should be a `RequestChain` with `wait` kwarg - previously CreateQuestionnaire handled the waiting but AddModelComponent does not wait on Questionnaire status which could cause confusion because a Questionnaire num_examples could be off when calling GetQuestionnaire right after adding model component.

Reply:
will make separate pr for this